src/app/components/publications/publication-properties/publication-properties.component.html

```diff
@@ -3,16 +3,18 @@
     <div class="col-lg-6 col-sm-12">
       <mat-card class="m-2">
         <app-text-input name="DOI"
-                        [(ngModel)]="publication.doi"
+                        [value]="publication.doi"
+                        (onChange)="onPropertyChanged($event, 'doi')"
                         (onSaveChanges)="onChangesSaved($event, 'doi')"></app-text-input>
       </mat-card>
     </div>
 
     <div class="col-lg-6 col-sm-12">
       <mat-card class="m-2">
         <app-text-input name="URL"
-                        [(ngModel)]="publication.url"
+                        [value]="publication.url"
```

Comment:
why do you sometimes use the property [value] and sometimes [(ngModel)]?

Reply:
[value] should be used everywhere, where we do not have any linking behaviour (like application areas or problem types). Because we do not want bi-directional linking since we want to check that the original object has changed.